vmicore/src/lib/os/windows/Constants.h

```diff
@@ -4,6 +4,7 @@
 namespace VmiCore::Windows
 {
     constexpr pid_t SYSTEM_PID = 4;
+    constexpr uint16_t winBuildRedstone3 = 17134;
```

Comment:
This build number corresponds to Redstone 4 not 3 https://www.vergiliusproject.com/kernels/x64/Windows%2010%20|%202016